app/controllers/ProdMonitoring.scala

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import actions.CustomActionBuilders
+
+import com.gu.tip.Tip
+import play.api.libs.circe.Circe
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ProdMonitoring(
+    actionBuilders: CustomActionBuilders,
+    components: ControllerComponents,
+    tipMonitoring: Tip
+)(implicit val ec: ExecutionContext) extends AbstractController(components) with Circe {
+
+  import actionBuilders._
+
+  def handleTipRequest(country: String, contributionType: String, paymentMethod: String): Action[AnyContent] =
```

Comment:
why is there handleTipRequest and sendTip? also is there a more specific word than "handle" can be used here?